editions/tw5.com/tiddlers/widgets/EventCatcherWidget.tid

```diff
@@ -7,16 +7,18 @@ type: text/vnd.tiddlywiki
 ! Introduction
 
 <<.from-version "5.1.23">>
-The event catcher widget traps JavaScript events dispatched within its child content, and allows invoking a series of ActionWidgets in response to the events. 
 
-In order for the events to be trapped they must:
+//This is a technically advanced widget intended for use by those familiar with HTML, CSS and DOM event handling.//
+
+The event catcher widget traps DOM-initiated Javascript events dispatched within its child content, and allows invoking a series of ActionWidgets in response to those events. 
 
-* be of one of the types specified as a parameter to the event catcher widget.
-* arise within a DOM node matching the selector specified as a parameter to the widget.
+In order for the events to be trapped they must:
 
-Use of the event catcher widget is beneficial when using large numbers of other trigger widgets such as the ButtonWidget is causing performance problems. The workflow it enables is akin to what is referred to as "event delegation" in JavaScript parlance. 
+* be of one of the events specified as a parameter to the event catcher widget's `events` attribute.
```

Comment:
Using both the terms parameter and attribute is confusing. Suggest dropping parameter, that is more consistent with other widget documentation.

Reply:
Agreed. I'd not have used "parameter" at all had it not been there to begin with.

Reply:
Writing consistent documentation gets tricky when working on multiple things. Filters have parameters and widgets have attributes.